tests/msg_recorder_test.cc

```diff
@@ -83,9 +82,17 @@ std::string MessageToStr(const TestMessage<T>& msg) {
     return serialized_msg;
 }
 
+RecorderConfigPtr RecorderTest::GetTestConfig() {
+    RecorderConfig temp_config;
+    temp_config.snapshot_intervals_ = {};
+    temp_config.interval_name_      = "";
+    temp_config.record_dir_         = GetTestTempDir();
+    return std::make_shared<RecorderConfig>(temp_config);
```

Comment:
nit:
```
return std::make_shared<RecorderConfig>({
    .record_dir = GetTestTempDir(),
    // 另外两个这种默认构造就是空，没必要专门写
});
```

Reply:
CI requires an initialization for `.snapshot_intervals_` so I gave an empty vector in the latest push